guides/doc-Managing_Hosts/topics/proc_integrating_satellite_with_red_hat_web_console.adoc

```diff
@@ -5,8 +5,8 @@ By default, Red Hat web console integration is disabled in {Project}. If you wan
 
 .Procedure
 
-* On {ProjectServer}, run `satellite-installer` with the  `--enable-foreman-plugin-remote-execution-cockpit` option:
+* On {ProjectServer}, run `{foreman-installer}` with the  `--enable-foreman-plugin-remote-execution-cockpit` option:
 +
 ----
-# satellite-installer --enable-foreman-plugin-remote-execution-cockpit
+# {foreman-installer} --enable-foreman-plugin-remote-execution-cockpit
```

Comment:
This one isn't rendering - the part of the script that adds "attributes" to subs= at the top must have missed this one as it doesn't have any options included at all.